src/components/base/suggester/Suggester.module.css

```diff
@@ -17,7 +17,8 @@
 .input_wrap .clear {
   position: absolute;
   right: 0;
-  top: var(--pt05);
+  top: 0;
+  bottom: 0;
```

Comment:
Er det her Suggesteren i søgning? Hvorfor skal den ændres?

Reply:
Den var forkert højde og bredde according to the design team